workers/unity/Assets/Playground/Scripts/Worker/WorkerConnectorBase.cs

```diff
@@ -142,19 +142,19 @@ protected virtual LocatorConfig GetLocatorConfig(string workerType)
                 catch (ConnectionFailedException e)
                 {
                     logger.HandleLog(LogType.Error,
-                        new LogEvent($"Failed attempt to create worker")
+                        new LogEvent($"Failed to create worker")
                             .WithField("WorkerType", workerType)
                             .WithField("Message", e.Message));
                     attempts--;
                 }
             }
 
             throw new ConnectionFailedException(
-                $"Exceeded maximum connection attempts ({attempts})",
+                $"Tried to connect {attempts} times - giving up.",
```

Comment:
Driveby: What will `attempts` be in this case? It feels like it will be 0.

This method should not be modifying the input parameter - can/should we lint for this (when we reenable linting...)? It's usually a code smell.

I think what'd be best is if it:
- rename the input parameter to `maximumAttempts` (or similar)
- copied the input parameter to be `remainingAttempts`

Reply:
Ah yes, I missed we were modifying the original input variable. Will clean up...